src/components/formatters/include/formatters/CFormatterJsonSDLRPCv1.h

```diff
@@ -174,11 +174,13 @@ int32_t formatters::CFormatterJsonSDLRPCv1::fromString(
   int32_t result = kSuccess;
 
   try {
+    Json::CharReaderBuilder builder;
+    const std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
     Json::Value root;
-    Json::Reader reader;
+    const int jsonLength = static_cast<int>(str.length());
```

Comment:
@ydementieiev 
- named wrongly according to SDL Coding Style

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)